File: Swift-Src/Swift-Src/pipe.swift
//
//  pipe.swift
//  Swift-Src
//
//  Created by Wesley Webb on 9/23/20.
//

import Foundation

class Pipe
{
    private var _fowardResult:Double;
    var forwardResult:Double {
        get { return self._fowardResult; }
        set { self._fowardResult = newValue; }
    }
    
    private var _backwardResult:Double;
    var backwardResult:Double {
        get { return self._backwardResult; }
        set { self._backwardResult = newValue; }
    }
    
    init()
    {
        self._fowardResult = 0.0;
        self._backwardResult = 0.0;
    }
}

File: Java-Src/Pipe.java
public class Pipe
{
    private double _forwardResult;
    private double _backwardResult;

    public Pipe()
    {
        this.setForwardResult(0.0);
        this.setBackwardResult(0.0);
    }

    public double getForwardResult() 
    {
        return this._forwardResult;
    }

    public void setForwardResult(double value) 
    {
        this._forwardResult = value;
    }

    public double getBackwardResult() 
    {
        return this._backwardResult;
    }

    public void setBackwardResult(double value) 
    {
        this._backwardResult = value;
    }
}
